utils/logalloc.cc

```diff
@@ -2129,7 +2129,7 @@ class reclaim_timer {
     const is_preemptible _preemptible;
     const size_t _memory_to_release;
     const size_t _segments_to_release;
-    tracker::impl& _tracker;
+    tracker::impl* _tracker;
 
```

Comment:
Why are we making it optional?

Reply:
@bhalevy ?

Reply:
Well, I don't understand subordinate timers either.